api/app/controllers/concerns/spree/api/v2/storefront/order_concern.rb

```diff
@@ -35,9 +35,8 @@ def find_spree_current_order
           end
 
           def supported_currencies
-            Spree::Config[:supported_currencies].split(',').map do |code|
-              ::Money::Currency.find(code.strip)
-            end
+            return spree_current_store.supported_currencies_list if spree_current_store.persisted?
+            [::Money::Currency.find(Spree::Config.currency)]
```

Comment:
```suggestion

```
there will be always store with supported currencies so we don't need this failsafe